test/service/test_manifest.py

```diff
@@ -1737,10 +1740,19 @@ def test_fetch_manifest(self, get_cached_manifest):
                     'Status': 302,
                     'Location': str(url),
                 }
-                if format_ == ManifestFormat.curl:
+                if format_ is ManifestFormat.curl:
                     expected_json['CommandLine'] = {
-                        'cmd.exe': f'curl.exe --location "{str(url)}" | curl.exe --config -',
-                        'bash': f"curl --location '{str(url)}' | curl --config -"
+                        'cmd.exe': f'curl.exe -L "{str(url)}" | curl.exe -K -',
+                        'bash': f"curl -L '{str(url)}' | curl -K -"
+                    }
+                else:
+                    if format_ is ManifestFormat.terra_bdbag:
+                        _, _, file_name = object_key.partition('manifests/')
```

Comment:
This suggests that the object key is not known. It is.

```suggestion
                        file_name = default_file_name
```